tests/query/PutUTest.cxxtest

```diff
@@ -0,0 +1,97 @@
+/*
+ * tests/query/PutUTest.cxxtest
+ *
```

Comment:
There already is an `tests/atoms/PutLinkUTest` -- its dangerous to have a second test, in a different directory, with a similar name